kit/layout/src/Skeleton/Skeleton.tsx

```diff
@@ -190,13 +195,21 @@ export function UISkeleton({
 
     const { isVisible, crossDissolveProgress } = useCrossDissolve(visible);
 
+    const containerStyles = React.useMemo(() => {
+        const containerStyles = [styles.container, styleProp];
+        if (persistStyles) {
+            return containerStyles;
+        }
+        return isVisible ? containerStyles : null;
+    }, [isVisible, persistStyles]);
```

Comment:
@savelichalex have just pointed out that this is a breaking change in API!

I'd better agree with Aleksey that the SkeletonView should remain to be a View without any style related logic.